runtime/binding-asyncapi/src/main/java/io/aklivity/zilla/runtime/binding/asyncapi/internal/config/AsyncapiServerNamespaceGenerator.java

```diff
@@ -52,7 +52,7 @@ public NamespaceConfig generate(
                 .inject(n -> injectTcpServer(n, servers, options, metricRefs))
                 .inject(n -> injectTlsServer(n, servers, options))
                 .inject(n -> injectProtocolRelatedBindings(n, servers, metricRefs))
-                .inject(n -> injectProtocolServers(n, servers, metricRefs))
+                .inject(n -> injectProtocolServers(n, binding.namespace, options, servers, metricRefs))
```

Comment:
This is correct to pass `binding.namespace`, but parameter is called `qname`, which is not consistent.

Suggest we add a transient `qname` field to `HttpAuthorizationConfig` and set during `AsyncapiNamespaceGenerator.init(BindingConfig)`, to specify the qualified name of the guard being used for authorization in the `binding` namespace.

Then this parameter can be removed from the call stack, and the `HttpAuthorizationConfig.qname` can be used where we originally attempted to use `HttpAuthorizationConfig.name`, but now `qname` will just work.

I'm planning to implement this generically in the engine so you wouldn't need to explicitly set `qname`, but it will help tidy the code if we do it this way for now and I can remove the line where you set `qname` when I'm adding the general approach later.